server/routes/incidentDetails/incidentDetailsPage.ts

```diff
@@ -701,14 +701,14 @@ const redirectToTaskList = (res: Response, draftId: number) => {
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const setUpRedirectForEditError = (res: Response, prisonerNumber: string, error: any, draftId: number) => {
   logger.error(`Failed to post edited incident details for draft adjudication: ${error}`)
+  // TODO I'm unsure this URL is correct - should it be /offence-code-selection/ ?
   res.locals.redirectUrl = `/offence-details/${prisonerNumber}/${draftId}`
```

Comment:
I think this one needs looking at - not sure where it's meant to be going as we don't have an /offence-details now (I think that was maybe the placeholder page I set up right at the start of MVP+1) - I'm guessing maybe it needs to go to /offence-code-selection/...

Reply:
Raise it as a bug - then the team can decide